examples/sequence_classification/mlflow_peft_no_lora_accelerate.py

```diff
@@ -75,15 +75,16 @@ def parse_args():
     parser.add_argument('--dataset_name', type=str, default='glue', help='The name of the Dataset (from the HuggingFace hub) to train on.')
     parser.add_argument('--cache_dir', type=str, default=None, help='Directory to read/write data.')
     parser.add_argument("--amp", type=str, choices=["bf16", "fp16", "no"], default="fp16", help="Choose AMP mode")
-    parser.add_argument("--optimize", type=str, default="AdamW", help="Choose the optimization computation method")
-
+    parser.add_argument("--optimizer", type=str, default="AdamW", help="Choose the optimization computation method")
+    parser.add_argument('--checkpoint_dir', type=str, default='/nas/test_case_set_{set#}/a100/{benchmark}/{task}/{model_name}/10epoch', help='Directory to save checkpoints.')
```

Comment:
Please refrain from using this specific path by default

Reply:
Agree. I have fixed it